tests/data/TestGroup.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace luya\testsuite\tests\data;
+
+final class TestGroup
```

Comment:
The test group should extend from https://luya.io/api/luya-cms-base-BlockGroup

Reply:
I completely agree with you. This should be done in others, too. See: 
- https://github.com/luyadev/luya-testsuite/blob/master/tests/data/TestBlock.php
- https://github.com/luyadev/luya-testsuite/blob/master/tests/cases/CmsBlockTestCaseTest.php

I can add it if you want..